resilience_stats/outage_simulator_LF.py

```diff
@@ -67,7 +67,7 @@ def simulate_outage(init_time_step, diesel_kw, fuel_available, b, m, diesel_min_
                     -load_kw / n_steps_per_hour * batt_roundtrip_efficiency,  # excess energy
                 )
         else:  # check if we can meet load with generator then storage
-            fuel_needed = (m * max(load_kw, diesel_min_turndown*diesel_kw) + b) / n_steps_per_hour
+            fuel_needed = (m * max(load_kw, diesel_min_turndown * diesel_kw) + b) / n_steps_per_hour
```

Comment:
Yes, excess generation is stored in the battery if there's room, otherwise excess generation is unused.

Reply:
@NLaws To be consistent with the recent changes and assumptions for CHP and diesel generator during an outage in a resilience run, which allows them to turn down to 0 kW but not curtail during an outage, I think we assume the diesel generator can turndown to 0.

Reply:
The min turn down is applied the same in repot.jl and the outage simulator from what I can see, so I'm not sure what you mean? (Unless the min turn down has been removed in one of the CHP branches?). Note that the min turn down for generator is zero, and thus it is zero in both the optimization and the outage simulator by default.

Reply:
@NLaws The change was made in chpbeta in PR #149 with the [MinTurndown Constraint](https://github.com/NREL/REopt_Lite_API/blob/d1023f5ffa4aa31eec5bfd39a6a8e6a5c41512f1/reo/src/reopt.jl#L335) only enforced over `p.TimeStepsWithGrid`. So yes, this was made in chpbeta and not develop (yet).

Reply:
Ok so do you want me to make changes in this branch to match chpbeta? Or do we deal with it when chpbeta merges into develop?

Reply:
I think make the change now/here so it's self-consistent with chpbeta since we're merging into chpbeta. Then both changes will get merged into develop in a month.